main.tf

```diff
@@ -8,29 +8,23 @@ locals {
 provider "random" {
```

Comment:
guess you can remove this then

Reply:
there is still the random_id usage for the final snapshot